### PR TITLE
[SRP] Expose lease container in Apr19 API.

### PR DESCRIPTION
**Pull request by huizlAzure**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by huizlAzure** [comment]
Previous PR for older API versions: https://github.com/Azure/azure-rest-api-specs/pull/5480/commits.

File: specification/storage/resource-manager/Microsoft.Storage/stable/2019-04-01/blob.json
Comment (nschonni) [inline specification/storage/resource-manager/Microsoft.Storage/stable/2019-04-01/blob.json]:
```suggestion
      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/blobServices/default/containers/{containerName}/lease": {
```

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5902 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5902/commits/002a7511212047e3f61f0176ebaaa65ce9c7509a

File: specification/storage/resource-manager/Microsoft.Storage/stable/2019-04-01/blob.json
Comment (huizlAzure) [reply specification/storage/resource-manager/Microsoft.Storage/stable/2019-04-01/blob.json]: Thanks! This is fixed in latest iterations.

**Review by sarangan12 — APPROVED** [review]

**Comment by sarangan12** [comment]
This PR introduces new path to an existing version of swagger. Per ARM guidelines, this requires ARM approval. So, I have added the required label and will wait for the ARM Feedback.

**Review by anthony-c-martin — APPROVED** [review]

**Pull request merged**
